apps/remote_control/lib/lexical/remote_control/build/error/location.ex

```diff
@@ -0,0 +1,71 @@
+defmodule Lexical.RemoteControl.Build.Error.Location do
```

Comment:
I can change position to return `Position`, and then rename the function that returns `Range` to `range()`. However, what should we name it when the result is a single integer?

Reply:
we would make it a range that selects the entire line. Start is on `{line, 0}`, end is on `{line + 1, 0}`

Reply:
But this logic is handled in this file:https://github.com/lexical-lsp/lexical/blob/main/apps/server/lib/lexical/convertibles/lexical.plugin.diagnostic.result.ex#L50C8-L50C25. 

I think it's reasonable to do this processing at that level because before that, we still need to perform operations like `uniq` and `refine`, We need to know whether the Elixir compiler is giving us the `line` or the `position`. It makes sense to keep it as a **single integer** instead of converting it.

Reply:
How about we handle the case of a range first?

https://github.com/lexical-lsp/lexical/pull/535/commits/6b7092e2b4d0dd2377b21c32625308cb5c0f36ee